src/views/Login.vue

```diff
@@ -1,57 +1,101 @@
-    </div>
-    <div class="input-container">
-      <label for="name" class="label">Email</label>
-      <input id="name" class="input" v-model="email">
-    </div>
-    <div class="input-container">
-      <label for="name" class="label">Password</label>
-      <input id="name" class="input" v-model="password" type="password">
-    </div>
-    <button :class="'primaryButton'" @click="printForm">Sign up</button>
+    <form @submit.prevent="onSubmit">
+      <div class="input-container">
+        <label for="name" class="label">First name</label>
+        <input id="firstName" v-model="firstName" class="input">
+      </div>
+      <div class="input-container">
+        <label for="lastName" class="label">Last name</label>
+        <input id="lastName" v-model="lastName" class="input">
+      </div>
+      <div class="input-container">
+        <label for="email" class="label">Email</label>
+        <input
+          id="email"
+          v-model="email"
+          :class="['input', ($v.email.$invalid && submitted) || !$v.email.email ? 'error-input' : '']"
+        >
+        <span class="error-label" v-if="!$v.email.email">El email no tiene un formato válido</span>
+        <span class="error-label" v-if="!$v.email.required && submitted">Campo requerido</span>
+      </div>
+      <div class="input-container">
+        <label for="password" class="label">Password</label>
+        <input
+          id="password"
+          type="password"
+          v-model="$v.password.$model"
+          :class="['input', $v.password.$error && ($v.password.required || submitted ) ? 'error-input': '']"
+        >
+        <span class="error-label" v-if="!$v.password.required && submitted">Campo requerido</span>
+        <span
+          class="error-label"
+          v-if="$v.password.$dirty && $v.password.required && !$v.password.hasNumber"
+        >El password debe contener al menos un número</span>
+        <span
+          class="error-label"
+          v-if="$v.password.$dirty && $v.password.required && !$v.password.hasUppercase"
+        >El password debe contener al menos una mayúscula</span>
+      </div>
+      <button class="primary-button">Sign up</button>
+    </form>
     <div class="horizontal-division"/>
-    <button :class="'secondaryButton'">Login</button>
+    <button class="secondary-button">Login</button>
   </div>
 </template>
 
 <script>
+import { required, email } from 'vuelidate/lib/validators'
+import { hasNumber, hasUppercase } from '@/utils/validations.js'
+
 export default {
-  name: "home",
+  name: 'home',
   components: {},
-  data() {
+  data () {
     return {
-      firstName: null,
-      lastName: null,
-      email: null,
-      password: null
-    };
+      firstName: '',
+      lastName: '',
+      email: '',
+      password: '',
+      submitted: false
+    }
+  },
+  validations: {
+    firstName: {
+      required
+    },
+    lastName: {
+      required
+    },
+    email: {
+      required,
+      email
```

Comment:
use another name for this like validEmailFormat

Reply:
The one in the line 73 is the specific argument I have to use according to Vuelidate documentation.

Reply:
![screenshot from 2019-01-14 22-51-48](https://user-images.githubusercontent.com/22583914/51153456-06f95f80-184f-11e9-8c38-c98b09c202ee.png)